src/shared/sam/activation.ts

```diff
@@ -82,24 +72,28 @@ export async function activate(activateArguments: {
     await resumeCreateNewSamApp()
 }
 
-async function registerServerlessCommands(params: {
-    extensionContext: vscode.ExtensionContext
-    awsContext: AwsContext
-    regionProvider: RegionProvider
-    channelLogger: ChannelLogger
-}): Promise<void> {
-    params.extensionContext.subscriptions.push(
+async function registerServerlessCommands(ctx: ExtContext): Promise<void> {
+    ctx.subscriptions.push(
         vscode.commands.registerCommand(
             'aws.samcli.detect',
             async () =>
                 await PromiseSharer.getExistingPromiseOrCreate('samcli.detect', async () => await detectSamCli(true))
         ),
+        vscode.commands.registerCommand('aws.lambda.getLambdaName', async () => {
```

Comment:
What's the point of this command? Is this a right-click action for the tree? If so, it doesn't need to be an activation event since it'll only be surfaced if the extension is activated.